ocf_datapipes/training/common.py

```diff
@@ -159,12 +167,23 @@ def open_and_return_datapipes(
         pv_datapipe = OpenPVFromNetCDF(
             pv=configuration.input_data.pv
         ).add_t0_idx_and_sample_period_duration(
-            sample_period_duration=timedelta(minutes=5),
+            sample_period_duration=timedelta(minutes=5),  # TODO Fix as India is 15 minutely
```

Comment:
There is a frequency entry in the config model which maybe we should use for this: https://github.com/openclimatefix/ocf_datapipes/blob/main/ocf_datapipes/config/model.py#L134

Reply:
Ah good shout, I'll update it to that, and the other ones as well